its/src/test/java/com/sonar/it/scanner/msbuild/sonarqube/MultiLanguageTest.java

```diff
@@ -296,7 +313,7 @@ void angular() {
 
   @Test
   void nonSdkFormat() {
-    assumeTrue(ORCHESTRATOR.getServer().version().isGreaterThan(9, 9)); // Multi-language unsupported in SQ99
+    assumeTrue(OSPlatform.isWindows() && ORCHESTRATOR.getServer().version().isGreaterThan(9, 9)); // Multi-language unsupported in SQ99
```

Comment:
If you want to disable tests in Windows or Linux go for annotations 
with EnableOnOS, DisableOnOS
https://junit.org/junit5/docs/5.2.0-RC1/api/org/junit/jupiter/api/condition/EnabledOnOs.html